.github/workflows/snyk.yml

```diff
@@ -0,0 +1,41 @@
+name: Snyk
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+  snyk:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@2541b1294d2704b0964813337f33b291d3f8596b # tag=v3
+      - uses: actions/setup-node@eeb10cff27034e7acf239c5d29f62154018672fd # tag=v3
+      - name: Prepare snyk
+        run: |
+          npm run setup:snyk
+          npm ci
+          yarn install --no-immutable --mode=skip-build
+      - name: Snyk Monitor
```

Comment:
Snyk Monitor is for security issues
Snyk Code is for static code analysis
Is Snyk Test for license issues ? 🤔

Reply:
**Monitor** uploads to the Snyk project to our Snyk App. (will run only on master)

**Test** is similar in functionality but doesn't upload to the Snyk app, it's used to "Upload result to GitHub Code Scanning" (runs all the time). This gives us those:

<img width="699" alt="Screen Shot 2022-07-14 at 9 39 08 AM" src="https://user-images.githubusercontent.com/4923043/178995800-b096d392-2475-4bd8-b86d-4cbd13f59603.png">